### PR TITLE
[NSH-2024] adds a sponsor and a new organizer

### PR DESCRIPTION
**Pull request by GeoffCraig**
Adds NetApp as a sponsor and Erica Williamson as an organizer

### CODE REVIEW

**Comment by phrawzty** [comment]
Hello @GeoffCraig. We can't merge this until your new organiser has confirmed their email address. If you want to merge the sponsor portion in the interim, you can wish to close this PR and make a unique PR for that modification, then open a new PR when the email confirmation is made. Your call.

**Comment by adrianmoisey** [comment]
Received an email, we are unblocked to merge now

File: static/img/sponsors/netappsfsxn.png
Comment (adrianmoisey) [inline static/img/sponsors/netappsfsxn.png:1]: This file is named incorrectly and the site is giving a 404
It should match that of the sponsor:
`netappfsxn`, without an "s" after "app"

Comment (GeoffCraig) [reply static/img/sponsors/netappsfsxn.png:1]: Sorry about, just fixed it and pushed the change.

**Pull request merged**